Error in main execution: ReferenceError: grip is not defined

=== Starting project scheduler ===
 Configuration: {projectReportId: 2681, workHoursReportId: 3283, parametersReportId: 3248, executorsReportId: 2777, taskDurationCode: 't3758', …}
 Loading message displayed to user
 --- Step 1: Fetching project data ---
 Project data URL: https://integram.io/orbits/report/2681?JSON_KV
   Fetching from URL: https://integram.io/orbits/report/2681?JSON_KV
   ✓ Received 25 items
 ✓ Fetched 25 items from project report
   - Template items: 11
   - Work items (В работе): 14
 --- Step 2: Fetching work hours configuration ---
 Work hours URL: https://integram.io/orbits/report/3283?JSON_KV
   Fetching from URL: https://integram.io/orbits/report/3283?JSON_KV
   ✓ Received 6 items
 ✓ Fetched 6 work hours settings
 --- Step 3: Fetching executors data ---
 Executors URL: https://integram.io/orbits/report/2777?JSON_KV
   Fetching from URL: https://integram.io/orbits/report/2777?JSON_KV
   ✓ Received 10 items
 ✓ Fetched 10 executors
 --- Step 4: Parsing work hours configuration ---
 Default work hours: {dayStart: 9, dayEnd: 18, lunchStart: 13}
   Processing setting: 2nd_op_tolerance = 15
   Processing setting: day_end = 18
   Processing setting: day_start = 9
   Processing setting: lunch_start = 13
   Processing setting: 1st_op_tolerance = 30
   Processing setting: 3rd_op_tolerance = 0
 ✓ Final work hours: {dayStart: 9, dayEnd: 18, lunchStart: 13}
   Work day: 9:00 - 18:00
   Lunch break: 13:00 - 14:00
 --- Step 5: Finding project start date ---
 Looking for "В работе" project with start date...
   Found project: "Установка витражей СПК (ОПЕРАЦИИ)"
   Start date string: "20.11.2025"
 ✓ Project: Установка витражей СПК (ОПЕРАЦИИ)
 ✓ Start date: 20.11.2025
 --- Step 6: Building template lookup ---
   Building template lookup map...
     Added operation normative: "Проверка доков на полноту и качество" = 120 min
     Added task parameters: "Проверка документации" = "115:849(-),2673:(4-)"
     Added operation normative: "Обработка документации от заказчика" = 240 min
     Added operation normative: "Получение реперных точек" = 180 min
     Added task parameters: "Подготовка (предоперации)" = "115:849(-),2673:(4-)"
     Added operation parameters: "Получение реперных точек" = "740:%(-)"
     Added operation normative: "Разметка витража на проекте" = 240 min
assignment.js:206     Added operation parameters: "Разметка витража на проекте" = "2673:(3-)"
assignment.js:172     Added operation normative: "Разработка последовательности сборки" = 180 min
assignment.js:206     Added operation parameters: "Разработка последовательности сборки" = "2673:(3-)"
assignment.js:172     Added operation normative: "Разгрузка стоек и ригелей" = 12 min
assignment.js:181     Added task executors: "Подготовка (предоперации)" needs 2 executors
assignment.js:189     Added operation executors: "Разгрузка стоек и ригелей" needs 2 executors
assignment.js:206     Added operation parameters: "Разгрузка стоек и ригелей" = "2673:(1-2)"
assignment.js:172     Added operation normative: "Сборка каркаса" = 40 min
assignment.js:181     Added task executors: "Монтаж каркаса" needs 2 executors
assignment.js:189     Added operation executors: "Сборка каркаса" needs 2 executors
assignment.js:172     Added operation normative: "Установка каркаса" = 30 min
assignment.js:189     Added operation executors: "Установка каркаса" needs 2 executors
assignment.js:172     Added operation normative: "Разгрузка стеклопакетов" = 15 min
assignment.js:181     Added task executors: "Подготовка к монтажу (предоперации)" needs 2 executors
assignment.js:189     Added operation executors: "Разгрузка стеклопакетов" needs 2 executors
assignment.js:206     Added operation parameters: "Разгрузка стеклопакетов" = "728:Подсобник(-),2673:(1-2)"
assignment.js:172     Added operation normative: "Монтаж стеклопакетов" = 30 min
assignment.js:181     Added task executors: "Монтаж стеклопакетов" needs 2 executors
assignment.js:189     Added operation executors: "Монтаж стеклопакетов" needs 2 executors
assignment.js:164     Added task normative: "Сдача выполненных работ" = 300 min
assignment.js:198     Added task parameters: "Сдача выполненных работ" = "1015:%(-),2673:(3-)"
assignment.js:212   Processed 11 template items
assignment.js:213   Summary: 1 task normatives, 10 operation normatives, 4 task executor specs, 5 operation executor specs, 3 task parameters, 5 operation parameters
assignment.js:1377 ✓ Template lookup built:
assignment.js:1378   - Tasks: 1
assignment.js:1379   - Operations: 10
assignment.js:1380   - Task executors: 4
assignment.js:1381   - Operation executors: 5
assignment.js:1382   - Task parameters: 3
assignment.js:1383   - Operation parameters: 5
assignment.js:1386 --- Step 7: Scheduling tasks and operations ---
assignment.js:792   Scheduling work items...
assignment.js:799   Found 14 work items to schedule
assignment.js:803   Initial scheduling time: 20.11.2025 09:00:00
assignment.js:821   --- Scheduling item 1/14: Operation "Проверка доков на полноту и качество" (2632) ---
assignment.js:365       Duration for "Проверка доков на полноту и качество" (2632): 120 min × 1 = 120 min [template]
assignment.js:1455 Error in main execution: ReferenceError: grip is not defined
    at assignment.js:883:20
    at Array.forEach (<anonymous>)
    at scheduleTasks (assignment.js:806:19)
    at HTMLDocument.main (assignment.js:1387:31)
main @ assignment.js:1455

--- a/CLAUDE.md
+++ b/CLAUDE.md
@@ -13,11 +13,3 @@ Your prepared branch: issue-25-ce19f963
 Your prepared working directory: /tmp/gh-issue-solver-1764870658409
 
 Proceed.
-
----
-
-Issue to solve: undefined
-Your prepared branch: issue-31-8cff7f39
-Your prepared working directory: /tmp/gh-issue-solver-1764876020289
-
-Proceed.

